tests/routes_tests/test_auth.py

```diff
@@ -100,6 +101,16 @@ def test_auth_me_jurisdiction_admin(
 def test_auditadmin_start(client: FlaskClient):
     rv = client.get("/auth/auditadmin/start")
     assert rv.status_code == 302
+    location = urlparse(rv.location)
+    query_vars = parse_qs(location.query)
+    assert query_vars["redirect_uri"]
+    redirect_uri = query_vars["redirect_uri"][0]
+
+    # common problem is a trailing slash on origin
+    # which makes a double slash like 'http://localhost//authorize'
+    # which won't work. So testing to make sure there is no '//'
+    # other than '://'
+    assert re.search("[^:]\/\/", redirect_uri) is None
```

Comment:
do we know what we expect the redirect_uri to be? could we just assert its that? or even just assert the whole query string?

Reply:
can't assert the whole query string, as there is random state in there.

asserting the redirect_uri might work but might also break if some test config changes later to not be based on `localhost`, making the test flakier than we need it to be.

I *think* this is the right balance?

Reply:
Do we have any test that the redirect_uri is the correct value (as opposed to just being well-formed)? That seems like something worth testing

Reply:
yeah, I'm adding that now.